android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -454,14 +454,15 @@ public View getView(int i, View view, ViewGroup viewGroup) {
         updateDebugView(pwPair, view);
         view.findViewById(R.id.ranging_debug_container).setVisibility(View.VISIBLE);
         view.findViewById(R.id.metadata_debug_container).setVisibility(View.VISIBLE);
-        mPwCollection.setPwsEndpoint(Utils.DEV_ENDPOINT);
+        mPwCollection.setPwsEndpoint(Utils.DEV_ENDPOINT, Utils.DEV_ENDPOINT_VERSION, "");
         UrlShortenerClient.getInstance(getActivity()).setEndpoint(Utils.DEV_ENDPOINT);
       } else {
         // Otherwise ensure it is not shown
         view.findViewById(R.id.ranging_debug_container).setVisibility(View.GONE);
         view.findViewById(R.id.metadata_debug_container).setVisibility(View.GONE);
-        mPwCollection.setPwsEndpoint(Utils.PROD_ENDPOINT);
-        UrlShortenerClient.getInstance(getActivity()).setEndpoint(Utils.PROD_ENDPOINT);
+        Utils.setPwsEndpoint(view.getContext(), mPwCollection);
+        UrlShortenerClient.getInstance(getActivity())
+          .setEndpoint(mPwCollection.getPwsEndpointUrl());
```

Comment:
This isn't always going to work.  Only the open source Pws does shortening.  You should leave the line as it was, pointing to PROD_ENDPOINT